clnt/client.go

```diff
@@ -0,0 +1,260 @@
+	JobId string
+}
+
+// takes parsed command line arguments and makes a Stop() RPC call to the server
+func (cc *StopCommand) run(c *kingpin.ParseContext) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	id := uuid.New()
+	if err := id.UnmarshalText([]byte(cc.JobId)); err != nil {
+		return err
+	}
+
+	resp, err := client.Stop(ctx, &pb.StopRequest{
+		JobId: id[:],
+	})
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(resp.Message)
+	return nil
+}
+
+func configureStopCommand(app *kingpin.Application) {
+	c := &StopCommand{}
+	cc := app.Command("stop", "Stop a job at remote server").Action(c.run)
+	cc.Flag("job", "Job ID of the job").Required().Short('j').StringVar(&c.JobId)
+}
+
+//  Status command
+type StatusCommand struct {
+	JobId string
+}
+
+// takes parsed command line arguments and makes a Status() RPC call to the server
+func (cc *StatusCommand) run(c *kingpin.ParseContext) error {
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	id := uuid.New()
+	if err := id.UnmarshalText([]byte(cc.JobId)); err != nil {
+		return err
+	}
+
+	resp, err := client.Status(ctx, &pb.StatusRequest{
+		JobId: id[:],
+	})
+
+	if err != nil {
+		return err
+	}
+
+	job_id := resp.GetJobId()
+	status := resp.GetStatus()
+	exitCode := resp.GetExitCode()
+
+	id = uuid.New()
+	if err = id.UnmarshalBinary(job_id); err != nil {
+		return err
+	}
+
+	if status == "running" {
+		fmt.Printf("Job %v: %v\n", id.String(), status)
+	} else {
+		fmt.Printf("Job %v: %v, Exit Code %v\n", id.String(), status, exitCode)
+	}
+	return nil
+}
+
+func configureStatusCommand(app *kingpin.Application) {
+	c := &StatusCommand{}
+	cc := app.Command("status", "Get status of a job at remote server").Action(c.run)
+	cc.Flag("job", "Job ID of the job").Required().Short('j').StringVar(&c.JobId)
+}
+
+//  Output command
+type OutputCommand struct {
+	JobId string
+}
+
+// takes parsed command line arguments and makes an Output() RPC call to the server
+func (cc *OutputCommand) run(c *kingpin.ParseContext) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	id := uuid.New()
+	if err := id.UnmarshalText([]byte(cc.JobId)); err != nil {
+		return err
+	}
+
+	stream, err := client.Output(ctx, &pb.OutputRequest{
+		JobId: id[:],
+	})
+
+	if err != nil {
+		return err
+	}
+
+	for {
+		resp, err := stream.Recv()
+		// if EOF, simply break the loop and return nil
+		if err == io.EOF {
+			break
+		}
+
+		// if err, return it
+		if err != nil {
+			return err
+		}
+
+		// for each received line, simply print it
+		for _, line := range resp.OutputLine {
+			fmt.Print(string(line))
+		}
+	}
+	return nil
+}
+
+func configureOutputCommand(app *kingpin.Application) {
+	c := &OutputCommand{}
+	cc := app.Command("output", "Get output of a job at remote server").Action(c.run)
+	cc.Flag("job_id", "Job ID of the job").Required().Short('j').StringVar(&c.JobId)
```

Comment:
You used `--job`, not `--job_id` on the other commands. It would be a little easier to use if the same flags were used across commands.